connectivity/ft_connectivity_corr.m

```diff
@@ -81,14 +80,13 @@
 % FiXME: If output is angle, then jack-knifing should be done
 % differently since it is a circular variable
 
-hasjack     = ft_getopt(varargin, 'hasjack', false);
-cmplx       = ft_getopt(varargin, 'complex', 'abs');
+hasjack     = ft_getopt(varargin, 'hasjack',  false);
+cmplx       = ft_getopt(varargin, 'complex',  'abs');
 feedback    = ft_getopt(varargin, 'feedback', 'none');
 dimord      = ft_getopt(varargin, 'dimord');
 powindx     = ft_getopt(varargin, 'powindx');
 pownorm     = ft_getopt(varargin, 'pownorm', 0);
-pchanindx   = ft_getopt(varargin, 'pchanindx');
-allchanindx = ft_getopt(varargin, 'allchanindx');
+pchan       = ft_getopt(varargin, 'pchanindx');
```

Comment:
here the variable is `pchan`, whereas the option is `pchanindx`. In the help you document the option `pchan`

Reply:
I see further in the code that the two are rather interchangeable. If you did not change the API, I suggest to keep the name `pchanindx`, it is more consistent with other `chanindx` options elsewhere

Reply:
this has been resolved I think